src/sonic-build-hooks/scripts/buildinfo_base.sh

```diff
@@ -72,7 +72,7 @@ set_reproducible_mirrors()
 
     local mirrors="/etc/apt/sources.list $(find /etc/apt/sources.list.d/ -type f)"
     for mirror in $mirrors; do
-        sed -i "$expression" "$mirror"
+        sed -i "$expression" "$mirror" || sudo sed -i "$expression" "$mirror"
```

Comment:
Can we change to run the command as root in docker-sonic-mgmt?

Reply:
how about now?